src/registrar/views/domain_request.py

```diff
@@ -321,9 +321,9 @@ def get(self, request, *args, **kwargs):
                     request,
                     "domain_request_intro.html",
                     {
-                        "hide_requests": True,
-                        "hide_domains": True,
-                        "hide_members": True,
+                        "hide_requests": False,
```

Comment:
**blocking question:** if we are setting hide requests to false  it doesn't look like there is any place where we are then setting it to true. It appears we are no longer ever hiding the nav bar values so with this being a feature that is removed this code should get deleted, both here and where it is then used in header_extended.html